test/aphroditeInterface_test.js

```diff
@@ -2,8 +2,20 @@ import { expect } from 'chai';
 import { StyleSheetTestUtils } from 'aphrodite';
 
 import aphroditeInterface from '../src/aphroditeInterface';
-
-describe('aphroditeInterface', () => {
+import aphroditeInterfaceNoImportant from '../src/no-important';
+
+const interfaces = [
+  {
+    name: 'aphroditeInterface',
+    interface: aphroditeInterface,
+  },
+  {
+    name: 'aphroditeInterfaceNoImportant',
+    interface: aphroditeInterfaceNoImportant,
+  },
+];
```

Comment:
I'm not a fan of this kind of testing. It makes understanding the tests and making changes more difficult. Instead, it would be better to have separate tests for each of the three files. Most of the tests will go in aphroditeInterfaceFactory, and the aphroditeInterface and aphroditeInterface/no-important tests will be very minimal.